docs/DEVELOP.md

```diff
@@ -1 +1,12 @@
-Developing this project requires access to a Kubernetes cluster and Go version 1.16 or later.
\ No newline at end of file
+Developing this project requires access to a Kubernetes cluster and Go version 1.16 or later.
+
+- working within a `codespace` you can install and instantiate a kind cluster for testing
+
+  - Follow instructions on `kind` website for install from binary.
+  - Move kind binary to `/usr/bin`:`mv ./kind /usr/bin`
```

Comment:
can we link to kind docs here for installation? folks might use a package manager so they don't necessarily need to move the binary